aiida_vasp/calcs/vasp.py

```diff
@@ -69,24 +69,17 @@ def define(cls, spec):
         super(VaspCalculation, cls).define(spec)
         # Define the inputs.
         # options is passed automatically.
-        spec.input('parameters', valid_type=get_data_class('dict'), help='The VASP input parameters (INCAR).', required=False)
+        spec.input('parameters', valid_type=get_data_class('dict'), help='The VASP input parameters (INCAR).')
         spec.input('dynamics',
                    valid_type=get_data_class('dict'),
                    help='The VASP parameters related to ionic dynamics, e.g. flags to set the selective dynamics',
                    required=False)
-        spec.input('structure',
-                   valid_type=(get_data_class('structure'), get_data_class('cif')),
-                   help='The input structure (POSCAR).',
-                   required=False)
+        spec.input('structure', valid_type=(get_data_class('structure'), get_data_class('cif')), help='The input structure (POSCAR).')
```

Comment:
This change is fine. In fact, not sure why this was not required in the first place. There is the NEB stuff, where `POSCAR`s sites in folders etc. but I think we should try to handle that outside of the calculation. I admit that I do not see how to do it right now, but if we need to revert this in the future we can do this. Right now, this is certainly required.

Reply:
Maybe it was there because of restarts...